codebuild_specs/e2e_workflow_base.yml

```diff
@@ -32,6 +32,10 @@ batch:
       buildspec: codebuild_specs/test.yml
       env:
         compute-type: BUILD_GENERAL1_LARGE
+    - identifier: integration_test
```

Comment:
i think this can be removed here, its duplicated further down; I remember adding this in the other PR last week